src/Compilers/CSharp/Test/Syntax/Parsing/MemberDeclarationParsingTests.cs

```diff
@@ -10477,5 +10470,3000 @@ public class Class
             }
             EOF();
         }
+
+        #region Missing > after generic
+
+        [Fact]
+        [WorkItem(24642, "https://github.com/dotnet/roslyn/issues/24642")]
+        public void MissingClosingAngleBracket_Method01()
+        {
+            foreach (var options in new[] { TestOptions.Script, TestOptions.Regular })
+            {
+                const string source =
+                    """
+                    static IEnumerable<(string Value, string Description) GetAllValues(Type t)
+                    {
+                        if (!t.IsEnum)
+                            throw new ArgumentException("no good");
+
+                        return Enum.GetValues(t).Cast<Enum>().Select(e => (e.ToString(), e.ToString()));
+                    }
+                    """;
+
+                UsingDeclaration(source, options,
+                    // (1,55): error CS1003: Syntax error, '>' expected
+                    // static IEnumerable<(string Value, string Description) GetAllValues(Type t)
+                    Diagnostic(ErrorCode.ERR_SyntaxError, "GetAllValues").WithArguments(">").WithLocation(1, 55)
+                    );
```

Comment:
```suggestion
                    Diagnostic(ErrorCode.ERR_SyntaxError, "GetAllValues").WithArguments(">").WithLocation(1, 55));
```

Reply:
General note, I found this style in other tests and also found it better because it's easier to paste the code from the result of a failed test run.